pennylane/ops/op_math/sprod.py

```diff
@@ -191,8 +191,13 @@ def terms(self):  # is this method necessary for this class?
         Returns:
             tuple[list[tensor_like or float], list[.Operation]]: list of coefficients :math:`c_i`
             and list of operations :math:`O_i`
+
```

Comment:
could add a separate pauli path like in https://github.com/PennyLaneAI/pennylane/pull/5287

Reply:
You can achieve it by `op.simplify().terms()`

Reply:
I think we're trying to move away from `.terms()` doing any simplification.

Reply:
yeah makes sense!

Reply:
Wait am I missing something, this version still simplifies stuff no?

```
op = 0.5 * X(0) + 0.7 * Z(1) + 0.1 * X(0)
op = 2. * op
op.terms(), type(op)

(([1.2, 1.4], [X(0), Z(1)]), pennylane.ops.op_math.sprod.SProd)
```

Reply:
That is because `Sum.terms()` still simplifies.

Reply:
Ah right 🙈